events_task/lib/my_app/content/content.ex

```diff
@@ -48,23 +35,49 @@ defmodule MyApp.Content do
     Repo.all(Confirmation)
   end
 
-  def list_user_confirmed_event_ids(user) do
+  def list_confirmations_by_user(id) do
+    from(c in Confirmation,
+      where: c.user_id == ^id,
+      select: c
+    )
+    |> Repo.all()
+  end
+
+  def list_confirmations_by_event(id) do
+    from(c in Confirmation,
+      where: c.event_id == ^id,
+      select: c
+    )
+    |> Repo.all()
+  end
+
+  def list_events_by_author_id(id) do
+    from(e in Event,
+      where: e.user_id == ^id,
+      select: e
+    )
+    |> Repo.all()
+  end
+
+  def list_events_confirmed(user) do
     if user do
-      from(c in Confirmation,
+      from(e in Event,
+        join: c in Confirmation,
```

Comment:
If you have associations setup in ecto schemas, you can use `assoc` helper instead of joining manually: `join: c in assoc(e, :confirmations)`